packages/api/trpc/router.ts

```diff
@@ -11,16 +13,16 @@ export const emailRouter = router({
     .mutation(async ({ input }: { input: { to: string } }) => {
       const result = await sendEmail(input.to);
       return {
-        success: result.success,
-        error: result.error,
-        code: result.code ? 'Code sent successfully' : undefined,
+        code: 'Code sent successfully',
       };
     }),
 });
 
 export const appRouter = router({
   email: emailRouter,
   auth: authRouter,
+  users: usersRouter,
```

Comment:
nitpick: and finally here create the router like this. If you like it you can do the change and you'll see it looks cleaner
```suggestion
  users: router(usersProcedures),
```